src/main/react-front-end/package.json

```diff
@@ -46,7 +46,7 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
   },
-  "proxy": "http://localhost:8080/",
+  "proxy": "http://localhost:80/",
```

Comment:
if we are now gonna use standard ports, do we need a proxy setting?

Reply:
Yes since frontend is still in port 3000.

Reply:
Makes sense to have it for ppl running UI separately out of the jar. Thanks!